cgp/utils.py

```diff
@@ -201,3 +201,25 @@ def primitives_from_class_names(primitives_str: Tuple[str, ...]) -> Tuple[Type[N
         primitives.append(primitives_dict[s])
 
     return tuple(primitives)
+
+
+def objective(
```

Comment:
i would remove this from this PR, we don't use it for keeping track of the number of objective function evaluations, do we?